### ISSUE TITLE
v2.0.0 and yarn pnp

### ISSUE THREAD

**Issue by Ayc0** [jamesknelson/mdx-loader#3]
I got this error when I install the v2.0.0 and start to run my project with yarn pnp

```
Error: Package "mdx-loader@2.0.0" (via "...") is trying to require the package
"loader-utils" (via "loader-utils") without it being listed in its dependencies

(@mdx-js/mdx, @mdx-js/tag, gray-matter, hast-util-to-html, hast-util-to-string,
mdx-constant, mdx-table-of-contents, prismjs, refractor, remark-emoji, remark-images,
remark-slug, remark-textr, typographic-base, unist-util-visit, mdx-loader)
```

**Comment by Ayc0** [comment]
I've checked and the package.json of the [release version of mdx-loader@2.0.0](https://unpkg.com/mdx-loader@2.0.0/package.json) doesn't correspond to the current package.json

**Comment by jamesknelson** [comment]
Sorry, I've moved this package over to the https://github.com/frontarm/mdx-util repository, and forgot to mark this as deprecated. I'll close this issue and re-open there.

**Comment by jamesknelson** [comment]
I've just published 2.0.1 which should fix this. Thanks for reporting it!
